app/Http/Controllers/ProductController.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Product;
+use Facade\FlareClient\Stacktrace\File;
+use Illuminate\Http\Request;
+
+class ProductController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view('upload-product');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $request->validate([
+            'name' => ['required'],
+            'description' => ['required'],
+            'price' => ['required'],
+            'category' => ['required'],
+            'path' => ['required','mimes:jpg,jpeg,gif,png,pdf']
+        ]);
+
+        $productModel = new Product;
+
+        if($request->file('path')){
+            $fileName = time().'_'.$request->file('path')->getClientOriginalName();
+            $filePath = $request->file('path')->storeAs('uploads', $fileName, 'public');
+
+            $productModel->name = $request->name;
+            $productModel->description = $request->description;
+            $productModel->price = $request->price;
+            $productModel->category = $request->category;
```

Comment:
Create a separate table for categories, and then add it as a foreign id attribute in the products table. You can seed them into the db or better yet, create a CategoryController for the categories CRUD operations. Then in the form, the categories will be added to a select input field with values as their IDs eg: 
`<option value="{{ $category->id }}">{{ $category->name }}</option>`. 
Then when you're creating new instances of the product, add categories as foreign IDs.